{"url":"https://api.github.com/repos/owncloud/core/issues/6063","labels_url":"https://api.github.com/repos/owncloud/core/issues/6063/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/6063/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/6063/events","html_url":"https://github.com/owncloud/core/issues/6063","id":23319502,"number":6063,"title":"owncloud 6 beta 5 linux client uploads 10mb max","user":{"login":"ruudschmahinda","id":6041609,"avatar_url":"https://gravatar.com/avatar/d6b63249cdac9fb301b91c301f4f9278?d=https%3a%2f%2fidenticons.github.com%2fba7968ef00f81ac2eb23f151ff1ccfa4.png&r=x","gravatar_id":"d6b63249cdac9fb301b91c301f4f9278","url":"https://api.github.com/users/ruudschmahinda","html_url":"https://github.com/ruudschmahinda","followers_url":"https://api.github.com/users/ruudschmahinda/followers","following_url":"https://api.github.com/users/ruudschmahinda/following{/other_user}","gists_url":"https://api.github.com/users/ruudschmahinda/gists{/gist_id}","starred_url":"https://api.github.com/users/ruudschmahinda/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/ruudschmahinda/subscriptions","organizations_url":"https://api.github.com/users/ruudschmahinda/orgs","repos_url":"https://api.github.com/users/ruudschmahinda/repos","events_url":"https://api.github.com/users/ruudschmahinda/events{/privacy}","received_events_url":"https://api.github.com/users/ruudschmahinda/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/bug","name":"bug","color":"fc2929"}],"state":"closed","assignee":null,"milestone":null,"comments":1,"created_at":"2013-11-26t14:56:36z","updated_at":"2013-11-26t15:32:27z","closed_at":"2013-11-26t15:32:27z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"i am using the owncloud client 4.2.18 in ubuntu 13.04 for an owncloud installation on an apache22 server in a freenas jail created by the owncloud 5.11 plugin.\r\n\r\nany file larger than 10 mb will not be uploaded. the errormessage reads:\r\n\"file not found.\"\r\n\r\nthe clients retries uploading the file again and again, always stopping at 10 mb.\r\n\r\ni created files increasing in size in my owncloud-sync folder using fallocate -l 12000000 test.dat (this file wasn't uploaded) but a file created with fallocate -l 11000000 test.dat will be uploaded.\r\n\r\nask for more details if needed. i am a bit new to this bug reporting.","closed_by":{"login":"deepdiver1975","id":1005065,"avatar_url":"https://gravatar.com/avatar/bdba071ed68da9ce8edae53c364902ef?d=https%3a%2f%2fidenticons.github.com%2f67e101ad9531b7d25697b76e503e062e.png&r=x","gravatar_id":"bdba071ed68da9ce8edae53c364902ef","url":"https://api.github.com/users/deepdiver1975","html_url":"https://github.com/deepdiver1975","followers_url":"https://api.github.com/users/deepdiver1975/followers","following_url":"https://api.github.com/users/deepdiver1975/following{/other_user}","gists_url":"https://api.github.com/users/deepdiver1975/gists{/gist_id}","starred_url":"https://api.github.com/users/deepdiver1975/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/deepdiver1975/subscriptions","organizations_url":"https://api.github.com/users/deepdiver1975/orgs","repos_url":"https://api.github.com/users/deepdiver1975/repos","events_url":"https://api.github.com/users/deepdiver1975/events{/privacy}","received_events_url":"https://api.github.com/users/deepdiver1975/received_events","type":"user","site_admin":false}}